src/components/pages/MyCases/MyCases.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './MyCases.less';
+import { Typography, Table, Button } from 'antd';
+
+//***There is a bug*** Currently, when you expand one caseObj they all expand. This may be an issue with accept and reject buttons - we don't want to accept all or reject all on accident!
```

Comment:
Great note on the bug. This will help future developers (or us) when we fine-tune the features.